src/test/java/com/cdancy/bitbucket/rest/features/RepositoryApiLiveTest.java

```diff
@@ -119,6 +119,18 @@ public void testListPermissionByGroup() {
         assertThat(permissionsPage.values()).isEmpty();
     }
 
+    @Test(dependsOnMethods = "testGetRepository")
+    public void testCreatePermissionByGroupNonExistent() {
+        boolean success = api().createPermissionsByGroup(projectKey, repoKey, "REPO_WRITE", randomString());
```

Comment:
I try with last bitbucket (5.0) and Bitbucket return this error : 

{
  "errors": [
    {
      "context": null,
      "message": "No such groups: 123",
      "exceptionName": null
    }
  ]
}